### DISCUSSION TITLE
[Feature] slug based sharing links

### DISCUSSION THREAD

**Discussion by lephtHanded** [immich-app/immich#6860] [Feature Request]
### The feature

currently shared album links look like this:
hxxps://domain.com/share/hdXLt9j-zJvvu-NTmjxM-TVgOOJytYp5Hp3iOTaO0ORCO3omoFPg3K9Em7VRuG3T6-0

[note-mark](https://github.com/enchant97/note-mark) utilizes 'slug' based links - so the link would have something *human readable* in it:
hxxps://domain.com/share/my-trip-to-annapolis-G3T6-0

if people prefer the fully randomized link style, maybe have a ENV value that can be set? 
`SHARELINK_TYPE=random|slug`

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Potentially a duplicate of #2724?

**Reply by lephtHanded** [reply]
oh - yeah, i thought that was related to the share URL host - i suppose you can close this one... i'll add these comments to that.
